README.md

```diff
@@ -124,6 +124,15 @@ $ curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/ni
 $ NIX_BUILD_GROUP_NAME=nixbuilder ./nix-installer install linux-multi --nix-build-group-id 4000
 ```
 
+## Upgrading Nix
+
+You can upgrade Nix (to the version specified [here](https://raw.githubusercontent.com/NixOS/nixpkgs/master/nixos/modules/installer/tools/nix-fallback-paths.nix)) by running:
+
+```
+sudo nix upgrade-nix
+```
+
+Alternatively, you can [uninstall](#uninstalling) and [reinstall](#usage). This method will install the version which was pinned in `src/settings.rs` at the time of compilation of the nix-installer.
```

Comment:
@Hoverbear @cole-h
I added a bit more information on which version will be installed on the second method. Feel free to suggest removing it if it is too much information for the README.